lib/templates/provision.sh.mustache

```diff
@@ -11,10 +11,10 @@ ssh-add -l
 . ./inventory/aws_keys
 
 # Tag any existing {{ app }} instances as being old
-ansible-playbook tag-old-nodes.yaml --limit tag_Environment_{{{ environment }}}
+ansible-playbook tag-old-nodes.yaml --limit tag_Environment_old
```

Comment:
I'm still not sure what we should call this tag... Giving the user control over it is a bit confusing, but `tag_Environment_old` is not the answer either...